main_functional_test.go

```diff
@@ -6,6 +6,7 @@
 // NOTE 1: You should probably have local instances of rabbit, kafka, etc. running
 // or  else the test suite will fail.
 
+//go:build functional
```

Comment:
We're still on go1.16 for building this, let's change back to `// +build functional`
https://github.com/batchcorp/plumber/blob/2058f18fef33b7fcfac1ae8668887475450b3697/go.mod#L3